README.md

```diff
@@ -206,12 +196,33 @@ Run integration tests on an OpenShift cluster:
 yarn run test-gui-openshift
 ```
 This will include the normal k8s CRUD tests and CRUD tests for OpenShift
-resources. It doesn't include ALM tests since it assumes ALM is not
-set up on an OpenShift cluster.
+resources.
+
+#### How the Integration Tests run in CI
+
+The end-to-end tests run against pull requests using [ci-operator](https://github.com/openshift/ci-operator/).
+The tests are defined in [this manifest](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml)
```

Comment:
The tests are generated by github.com/openshift/release/ci-operator/config/openshift/console/openshift-console-master.yaml using ci-operator-prowgen (https://github.com/openshift/ci-operator-prowgen) into https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml